Add design history on preparing for GP IT integration (#462)
NHSDigital/prevention-services-design-history: app/breast-screening-pathway/2026/04/before-integrating-with-GP-IT/index.md

---

> title: Before we can build an integration with GP patient records

Reviewer: The title seems a bit strange to me, as if it's only half a sentence?

How about

"What we need to do before we can integrate with GP records"

or 

"What we need in order to integrate with GP records"

or something snappier?

Author: Thanks for the suggestion! Going with "4 things to do before integrating breast screening with GP records"

description: We want to send breast screening results directly to GP patient records. Here's what we need to work through first.
date: 2026-04-16
tags:
  - GP IT
  - SNOMED CT
author:
  - Veronika Jermolina
opengraphImage:
  src: /breast-screening-pathway/2026/04/before-integrating-with-GP-IT/GP-IT-plan.png
  alt: 4 steps to consider before integrating with GPs
---

> Each year 3 million paper result letters are sent from breast screening offices (BSOs) to general practices (GPs) to update patient records. About a year ago we ran a discovery to [understand if we could solve this problem nationally](https://design-history.prevention-services.nhs.uk/explore-team/2025/06/insights-and-opportunities-for-sharing-breast-screening-results-with-gp-surgeries/).

Reviewer: GP is one of the few acronyms that doesn't need spelling out! - see https://service-manual.nhs.uk/content/a-to-z-of-nhs-health-writing#g

Author: Did not know that - thank you!

> Each year 3 million paper result letters are sent from breast screening offices (BSOs) to general practices (GPs) to update patient records. About a year ago we ran a discovery to [understand if we could solve this problem nationally](https://design-history.prevention-services.nhs.uk/explore-team/2025/06/insights-and-opportunities-for-sharing-breast-screening-results-with-gp-surgeries/).

Reviewer: Make the opening para a little more punchy?

```suggestion
3 million paper result letters are sent by breast screening offices (BSOs) to general practices (GPs) every year. A year ago we ran a discovery to [understand if we could solve this problem nationally](https://design-history.prevention-services.nhs.uk/explore-team/2025/06/insights-and-opportunities-for-sharing-breast-screening-results-with-gp-surgeries/).
```

The reality was more complex than we imagined.

The legacy NBSS system runs in 75 local instances - we'd need to implement a solution in every BSO. The patient record is managed by 2 dominant GP IT providers – who would need to make changes, adding a dependency on their delivery roadmaps.

So even a medium-term solution would be unpredictable, costly and time-consuming.

Our discovery exposed the limitations of NBSS and helped inform a decision to replace it with a modern breast screening service.

![4 steps to consider before integrating with GPs](GP-IT-plan.png)

## New breast screening service pilot

> Almost a year since then, team [Manage breast screening](../../../../manage-breast-screening/) launched a live pilot of the first slice of the new breast service in Humberside - our pilot partners. We are months away from being able to generate results in the new breast screening service.

Reviewer: No need to repeat the time period?

```suggestion
Since the discovery, team [Manage breast screening](../../../../manage-breast-screening/) have launched a live pilot of the first slice of the new breast service in Humberside - our pilot partners. We are months away from being able to generate results in the new breast screening service.
```

This means we are no longer held back by getting data out of NBSS and in theory, can move forward. However, before integration work can start, we need to work through some challenges. 

## Decide what outcomes to record

We already know that we need to record the following outcomes, or screening results:

- normal (routine recall): participant invited again in 3 years
- did not attend: participant didn't attend and will be invited again
- technical recall: mammogram not of sufficient quality, participant invited back
- recall for assessment (abnormal result): further investigation needed

> We still need to consider whether new outcomes, like [partial mammography](../../../../manage-breast-screening/2026/02/improving-the-partial-mammogram-process/), are needed and agree the full list with the delivery teams. We should also consider what action, if any, should GPs take on receiving them.

Reviewer: Bit unclear what the "them" refers to here?

```suggestion
We still need to consider whether new outcomes, like [partial mammography](../../../../manage-breast-screening/2026/02/improving-the-partial-mammogram-process/), are needed and agree the full list with the delivery teams. We should also consider what action, if any, should GPs take when receiving the information.
```

## Describe and sequence approval steps

As this work relies on GP IT systems, there are various approval steps and gateways that we'll need to go through. Last time we looked into this problem, some of these steps were:

- Digital services for integrated care (DSIC) front door
- Joint GP IT Liaison (JGPITL)
- Joint GP IT Committee (JGPITC)

> - GP IT roadmap

Reviewer: Are there any links for these things that can be added?

Author: Removed Joint GP IT Liaison (JGPITL), added links to DSIC and JGPITC

We need to confirm if these are still the right steps and the order in which we should go through them, and if there are any dependencies.

> We are speaking to other teams who we can collaborate with and learn from. One of those is the PDM (Patient data manager) team, who are simplifying integration with GP IT systems. Another is the [Health check online team](../../../../nhs-health-check-online/), who are piloting a solution with [PDM](https://digital.nhs.uk/services/patient-data-manager).

Reviewer: Move the link for PDM up to the first usage, and put the abbreviation in brackets?

```suggestion
We are speaking to other teams who we can collaborate with and learn from. One of those is the [Patient data manager](https://digital.nhs.uk/services/patient-data-manager) (PDM) team, who are simplifying integration with GP IT systems. Another is the [Health check online team](../../../../nhs-health-check-online/), who are piloting a solution with PDM.
```

## Choose SNOMED CT codes

If we wanted to automate the processing of breast screening results, we'd need to tag them with machine-readable codes that GP IT systems can understand and act upon. For example:

- a normal breast screening result can be automatically filed to the patient record

> - an abnormal result can be flagged to GPs, who can decide what action to take - such as offering the person support while they go through the assessment

Reviewer: ```suggestion
- an abnormal result can be flagged to GPs, who can decide what action to take, such as offering the person support while they go through the assessment
```

> There are already SNOMED CT (Systematised Nomenclature of Medicine - Clinical Terms) codes to denote breast screening outcomes, the most used codes being:

Reviewer: Add a link to a page about what SNOMED-CT is generally?

```suggestion
There are already [SNOMED CT](https://digital.nhs.uk/services/terminology-and-classifications/snomed-ct) (Systematised Nomenclature of Medicine - Clinical Terms) codes to denote breast screening outcomes, the most used codes being:
```
